### DISCUSSION TITLE
Rendering problems under multiple cameras

### DISCUSSION THREAD

**Discussion by BoBoBoram37** [crest-ocean/crest#1088] [Q&A] [answered]
I added several cameras in the scene at different positions and angles to detect the state of different planes of objects. At this time, cameras seem to interact with each other, rendering the color of the ocean on the surface of objects
Unity2019.4.10f  built-in pipeline   crest 4.9
![screenshot](https://user-images.githubusercontent.com/116548150/197514914-7ff7803e-dfbf-4efd-a49d-2d76eb9e8883.png)

**Comment by BoBoBoram37** [comment]
@daleeidd  @huwb Any suggestions? please

**Comment by magenta404** [comment, marked as answer]
Crest currently only supports one active camera at a time.
